src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/SagaTransactionResource.java

```diff
@@ -27,6 +29,7 @@
 
 @RestController
 @RequestMapping("/api/internal")
+@LepService("service.internal")
```

Comment:
@RomanTeren adding LEPs on the REST level is not recommended. 
is there is a possibility to add LEP on the Service level instead?